client/recipient.go

```diff
@@ -94,6 +95,11 @@ type TemplateVariable struct {
 	Default string `json:"default_value"`
 }
 
+type WebhookHeader struct {
+	Key   string `json:"header"`
```

Comment:
```suggestion
	Name   string `json:"name"`
```
Should this be name instead of Key and header?

Reply:
I copied this from our external package struct but was wondering similarly if we wanted to switch it over. Personally for me, `Key` is clearer when speaking about HTTP headers because that's what things like Postman etc use but not a super strongly held opinion

Reply:
Ahh gotcha that's my bad then, matching the external struct is definitely what we want. I don't have a strong opinion on it and we can change it if needed... Names are hard